pkg/kine/server/update.go

```diff
@@ -31,10 +32,28 @@ func (l *LimitedServer) update(ctx context.Context, rev int64, key string, value
 	)
 
 	if rev == 0 {
+		ctx, span := tracer.Start(ctx, "backend.create")
+		defer span.End()
+		span.SetAttributes(
+			attribute.String("key", key),
+			attribute.Int64("lease", lease),
+		)
 		rev, err = l.backend.Create(ctx, key, value, lease)
+		//TODO: why are here no error checks?
 		ok = true
+		span.SetAttributes(attribute.Bool("ok", ok))
+		createCnt.Add(ctx, 1)
 	} else {
+		ctx, span := tracer.Start(ctx, "backend.update")
+		defer span.End()
+		span.SetAttributes(
+			attribute.String("key", key),
+			attribute.Int64("revision", rev),
+			attribute.Int64("lease", lease),
+		)
 		rev, kv, ok, err = l.backend.Update(ctx, key, value, rev, lease)
+		span.SetAttributes(attribute.Bool("ok", ok))
+		updateCnt.Add(ctx, 1)
```

Comment:
You sometimes increase the counter before performing the action and somtimes afterwards. I see that this would not make a difference here *because there is no way to return early) but let's be consistent.